File: NumberOf1Bits/Solution.py
class Solution(object):
    def hammingWeight(self, n):
        """
        :type n: int
        :rtype: int
        """
        mask = 0x0001
        i = 0
        num = 0
        while i < 32:
            if (n & mask) != 0:
                num += 1
            mask = mask << 1
            i += 1
        return num

File: NumberOf1Bits/Solution.c
int hammingWeight(uint32_t n) {
    int num = 0;
    int mask = 0x1;
    for (int i = 0; i < 32; i++) {
        if ((n & mask) != 0) num++;
        mask <<= 1;
    }
    return num;
}